plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ServletClasspathProvider.java

```diff
@@ -30,7 +30,7 @@ public ServletClasspathProvider() {
 
   @Override
   public IClasspathEntry[] resolveClasspathContainer(IRuntime runtime) {
-    java.nio.file.Path cloudSdkPath = new CloudSdkProvider().getCloudSdkLocation().toPath();
+    java.nio.file.Path cloudSdkPath = new CloudSdk.Builder().build().getSdkPath();
```

Comment:
We seem to be repeating 
new CloudSdk.Builder().build() 
and 
new CloudSdk.Builder().build().getSdkPath()
a lot. Not in this PR, but any chance we could wrap one or both of these into a static method in CloudSdk? E.g CloudSdk.getDefaultSdk() and CloudSdk.getDefaultSdkPath()?